service/user/user_controller.py

```diff
@@ -0,0 +1,23 @@
+from flask import Blueprint, jsonify
+from service.user.user_service import auth
+
+user_controller = Blueprint('user', __name__)
+
+
+@user_controller.route('/<user_name>', methods=['GET'])
+@auth.login_required
+def get_user(user_name):
+    """
+    Retrieves the user information for the given user name.
```

Comment:
The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

:-P

Reply:
Hatte ich schon vor unserem Gespräch heute morgen geschrieben :)

Reply:
Ist aber nicht nur der eine docstring, kannst bei Gelegenheit auch die anderen noch überarbeiten.